tests/foreman/cli/test_errata.py

```diff
@@ -1125,37 +1126,44 @@ def test_positive_check_errata_dates(module_entitlement_manifest_org, module_tar
 
     :BZ: 1695163
     """
-    product = module_target_sat.api.Product(organization=module_entitlement_manifest_org).create()
+    product = module_target_sat.api.Product(organization=module_sca_manifest_org).create()
     repo = module_target_sat.cli_factory.make_repository(
         {'content-type': 'yum', 'product-id': product.id, 'url': REPO_WITH_ERRATA['url']}
     )
     # Synchronize custom repository
     module_target_sat.cli.Repository.synchronize({'id': repo['id']})
     result = module_target_sat.cli.Erratum.list(options={'per-page': '5', 'fields': 'Issued'})
-    assert 'issued' in result[0]
+    result = result.split()
+    assert 'Issued' in result[0]
```

Comment:
```suggestion
    assert 'Issued' in result.split()[0]
```

Reply:
or `result = module_target_sat.cli.Erratum.list(options={'per-page': '5', 'fields': 'Issued'}).split()`?

Reply:
I had separated the steps for clarity, but we can certainly 
split directly from `cli.Erratum.list` command.